cmds/core/sshd/sshd.go

```diff
@@ -63,13 +63,15 @@ func runCommand(c ssh.Channel, p *pty.Pty, cmd string, args ...string) error {
 		ps, _ = p.C.Process.Wait()
 	} else {
 		e := exec.Command(cmd, args...)
-		e.Stdin, e.Stdout, e.Stderr = c, c, c
+		e.Stdout, e.Stderr = c, c
```

Comment:
now the problem is that if I do this
echo hi | ssh machine dd 
it will not work as there is no stdin ...